### DISCUSSION TITLE
Is it already present or in the future plans an "invalidate" helper?

### DISCUSSION THREAD

**Discussion by frederikhors** [jycouet/kitql#112] [Q&A]
Is it already present or in the future plans an "invalidate" helper like:

1. https://formidable.com/open-source/urql/docs/api/graphcache/#invalidate or
1. https://react-query.tanstack.com/guides/query-invalidation?

This is a game changer for me.

**Comment by frederikhors** [comment]
I'm asking this because I'm getting from websockets resources to "invalidate" or simply "re-query" which are in many stores.

Using urql or apollo is simpler because they have a normalized global cache.

What do you think, @jycouet?

**Comment by jycouet** [comment]
Today you can use `.resetCache()` which is exactly what I use when I want to invalidate data.
It's already there, you can just use it :)
